support/getting-started/engagement-tracking.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Getting Started with Engagement Tracking"
+description: "This guide outlines some of the different ways you can do engagement tracking with SparkPost"
+---
+
+With SparkPost, there are a multitide of ways to configure engagement tracking with our customers.  This guide will outline some of the initial options, and provide references to other guides for configuring custom tracking domains and different ways to use **HTTPS** with your custom tracking domains.
+
+## Getting Started
+Customers can view their engagement through the use of SparkPosts Events.  These events can be searched and viewed in many different ways.  They can be viewed within [SparkPost Signals](https://www.sparkpost.com/docs/reporting/signals-analytics/), queried and searched via the [Events API](https://developers.sparkpost.com/api/events/) or sent to your own environments through the use of [Event Webhooks](https://developers.sparkpost.com/api/webhooks/) to name a few.
+
+
+### Notes on Click Tracking
+SparkPost click tracking links are highly dependent on the size of the original link supplied within the REST Transmission or SMTP request.  If there are concerns over link length within an email with SparkPost's click tracking enabled, first verify the original link length.
+
+> Link sizes of over **4096** bytes will be rejected on redirection
+
+## REST Transmissions with Engagement Tracking
+Transmissions sent via the SparkPost Transmissions API have engagement tracking on by default for all customers.  This includes a top open pixel, and bottom open pixel, and click tracking on all links within the email content.  These options can be configured via [Transmissions API](https://developers.sparkpost.com/api/transmissions/) level options.  For example, if you wish to use open tracking, but not click tracking, you can set the following options in your JSON payload for a REST transmission
```

Comment:
`This includes a top open pixel, and bottom open pixel, and click tracking on all links within the email content.` - Could probably remove the `and` before `bottom open pixel`. NBD, though.

Line below (29) has a trailing comma that could probably be removed, as well.
```
"ip_pool": "my_ip_pool",
```

Reply:
Ha yep thanks.